dist/common/scripts/scylla-housekeeping

```diff
@@ -71,13 +74,16 @@ def get_json_from_url(path):
     # to enforce a wallclock timeout.
     result = pool.apply_async(get_url, args=(path,))
     try:
-        retval = result.get(timeout=5)
+        status, retval = result.get(timeout=5)
     except mp.TimeoutError as err:
         pool.terminate()
         pool.join()
         raise
+    if status == 1:
+        raise RuntimeError(f'Failed to get "{path}" due to the following error: {retval}')
     return json.loads(retval)
```

Comment:
Why not put an except clause here instead?

Reply:
as mentioned above, the whole reason for this fix, is an Exception which isn't serializable, hence except clause would be much use in that case.

Reply:
@avikivity , in addition to what israel (@fruch) said, catchig exception there, you won't get legit error, instead of it you will get serialization error.